rt/frontend/jaxrs/src/test/java/org/apache/cxf/jaxrs/utils/JAXBUtilsTest.java

```diff
@@ -45,7 +48,10 @@ public void extendedXmlJavaTypeAdapter() {
     }
 
     private void correctValueType(Class<?> clazz) {
-        Field field = clazz.getDeclaredFields()[0];
+        Field[] fields = clazz.getDeclaredFields();
```

Comment:
@prathyushreddylpr the `correctValueType(...)` is called with two classes: CustomerDetailsWithSimpleAdapter & CustomerDetailsWithExtendedAdapter, both have a single field. Where other fields are coming from?

Reply:
@reta , When we call correctValueType() with class CustomerDetailsWithExtendedAdapter we get two fields. The second field is "org.apache.cxf.jaxrs.utils.JAXBUtilsTest$CustomerDetailsWithExtendedAdapter.this$0" and we are getting this because CustomerDetailsWithExtendedAdapter is an inner class of JAXBUtilsTest. Since it's not static, it holds an implicit reference to the JAXBUtilsTest instance that created it, which is represented as a synthetic field this$0.

Reply:
Thanks @prathyushreddylpr , this is a good finding, I don't see any reasons why this classes are not declared as static, could you please change the declaration for those? thank you

Reply:
@reta  Updated the classes to be static

Reply:
Thank you